_i18n/es/resources/user-guides/ledger-wallet-cli.md

```diff
@@ -1,165 +1,165 @@
 
-6. You may have to hit confirm twice before it proceeds.
+6. Puedes tener que hacer clic en confirmar dos veces antes de proceder.
 
-7. Your Ledger Monero wallet will now be generated. Note that this may take up to 5-10 minutes. Furthermore, there will be no immediate feedback in the CLI nor on the Ledger.
+7. Tu monedero Ledger Monero será ahora generado. Ten en cuenta que esto puede tomar de 5 a 10 minutos. Además, no habrá realimentación inmediata en la consola de comandos o en la Ledger.
 
-8. `monero-wallet-cli` will start refreshing. Wait until it has fully refreshed.
+8. `monero-wallet-cli` comenzará a refrescarse. Espera a que se haya refrescado totalmente.
 
-Congratulations, you can now use your Ledger Monero wallet in conjunction with the CLI.
+Felicidades, ahora puedes utilizar tu monedero Ledger Monero en conjunto con la consola de comandos.
 
-### 4. A few final notes
+### 4. Algunas notas finales
 
-1. We'd strongly advise to test the full process first. That is, send a small amount to the wallet and subsequently restore it (using aforementioned guide) to verify that you can recover the wallet. Note that, upon recreating / restoring the wallet, you ought to append the `--restore-height` flag (with a block height before the height of your first transaction to the wallet) to the command in step 3 (Windows), step 5 (Mac OS X), or step 3 (Linux). More information about the restore height and how to approximate it can be found [here](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
+1. Recomendamos fuertemente probar el proceso completo primero. Esto es, enviar una pequeña cantidad al monedero y después restaurarlo (utilizando la guía ya mencionada) para verificar que puedes recuperar el monedero. Ten en cuenta que, al recrear / restaurar el monedero, deberás adjuntar la bandera `--restore-height` (con una altura de bloque anterior a la altura de tu primera transacción al monedero) al comando en el paso 3 (Windows), paso 5 (Mac OS X), o paso 3 (Linux). Más información acerca de la altura de restauración y cómo aproximarla puede encontrarse [aquí](https://monero.stackexchange.com/questions/7581/what-is-the-relevance-of-the-restore-height).
 
-2. If you use a remote node, append the `--daemon-address host:port` flag to the command in step 3 (Windows), step 5 (Mac OS X), or step 3 (Linux).
+2. Si utilizar un nodo remoto, adjunta la bandera `--daemon-address host:port` al comando en el paso 3 (Windows), paso 5 (Mac OS X), o paso 3 (Linux).
```

Comment:
The word "utilizar" translated in the text is in past tense, as in "used"; utilizar, utilizado. I think it meant "utiliza" or "utilizas", because it has a bad gramatical structure if we leave it "utilizar = used".

"Si **utiliza** un nodo remoto,..." or "Si **utilizas** un nodo remoto,..."

Reply:
Where is says "flag" in the source, it should say "option" (see #827)
You'll have to decide whether to change 'bandera' to 'opción', I'm not sure which is more frequent in Spanish

Reply:
In some way I agree in the term "opción", but in this case is a "flag" ("marca" as in "a signal"; without the sign, it won't "run"; "bandera"). I would vote to keep it as "bandera", with this arguments. 
In the #827 issue the "option = opción" word, also, "wallet option = opción del monedero", are pointing out the options you are able to use in the command line. That's what I understand, but when you use a flag, you need to give a "special" sign in order for the CLI to run. What do you think?

Reply:
[Microsoft Terminology Search](https://www.microsoft.com/en-us/language/Search?&searchTerm=flag&langID=Spanish&Source=true&productid=All) says **"Marca"**.

Broadly, a marker of some type used by a computer in processing or interpreting information; a signal indicating the existence or status of a particular condition. Flags are used in such areas as communications, programming, and information processing.

Reply:
If we change it for the word "opción" it will change the whole meaning of what it really is.

Reply:
I'm not sure what you mean, but "--daemon-address" is one of the command line options of the monero-wallet-cli program, see the other PR and
https://en.wikipedia.org/wiki/Command-line_interface#Command-line_option

I can't post a link like you do but here's the MS term entry:

![image](https://user-images.githubusercontent.com/15184875/43878341-db2734a4-9b9e-11e8-83e4-20334c38c53a.png)

Reply:
"Option" is not "flag".

Reply:
Wikipedia sometime is a terrible source of information. Take a look at this [wikipedia flag Spanish definition](https://es.wikipedia.org/wiki/Flag).

Reply:
I can agree with what you said about Wikipedia, as I mentioned right now in a past comment. And for the original comment, yes, I used a wrong conjugation of the verb, I'm correcting it right now.

And for the other, I haven't seen the other PR for today but I can use _marcador_ since it's a more common word and I think it's more related to _flag_ than "opción" or "bandera".